src/components/content-tab-assistant.tsx

```diff
@@ -73,31 +84,81 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 		}
 	}, [ messages ] );
 
+	useEffect( () => {
+		if ( isAuthenticated && inputRef.current ) {
+			inputRef.current.focus();
+		}
+	}, [ isAuthenticated ] );
+
+	const disabled = isOffline || ! isAuthenticated;
 	return (
 		<div className="h-full flex flex-col bg-gray-50">
-			<div data-testid="assistant-chat" className="flex-1 overflow-y-auto p-8">
-				<div className="text-gray-500 mb-4">
-					Welcome to the Studio assistant. I can help manage your site, debug issues, and navigate
-					your way around the WordPress ecosystem.
-				</div>
-				<div>
-					{ messages.map( ( message, index ) => (
-						<Message key={ index } isUser={ message.role === 'user' }>
-							{ message.content }
-						</Message>
-					) ) }
-					<div ref={ endOfMessagesRef } />
-				</div>
+			<div
+				data-testid="assistant-chat"
+				className={ cx(
+					'flex-1 overflow-y-auto px-8 py-4',
+					! isAuthenticated && 'flex items-end'
+				) }
+			>
+				{ isAuthenticated ? (
```

Comment:
I wonder if it would be more readable to have the markup for authenticated vs unauthenticated be in separate components or at least in separate methods within this component. I struggle to read this ternary statement that spans almost 50 lines.

Reply:
I agree reducing the breadth of the ternary would improve readability. As also referenced in https://github.com/Automattic/studio/pull/189#discussion_r1621528770, the intent is to have a broader component reorganization once the Assistant feature grows with future UI updates. I think this change is good to merge for the time being, to address in upcoming UI PRs.